core/src/v2/schema.rs

```diff
@@ -287,8 +289,6 @@ pub trait Apiv2Schema {
 }
 
 impl Apiv2Schema for () {}
```

Comment:
I'm not sure about this one also.
Don't see any use case and also in the other hand, the generated OpenApi json may be wrong!

There is also no test for this line.